controllers/cluster.js

```diff
@@ -4,6 +4,7 @@ const Node = require('../server/models').Node;
 const Database = require('../server/models').Database;
 const Config = require('../server/models').Config;
 const Machine = require('docker-machine');
+const Sequelize = require('sequelize');
```

Comment:
I assume this can be removed since you ended up filtering the objects after retrieval?

Reply:
Oh yes, you're right.
Great catch!

Reply:
Removed the line.